django/db/backends/oracle/base.py

```diff
@@ -450,7 +450,7 @@ def _param_generator(self, params):
         else:
             return [p.force_bytes for p in params]
 
-    def _fix_for_params(self, query, params):
+    def _fix_for_params(self, query, params, handle_as_dict=False):
```

Comment:
I'd rename this parameter to "unify_by_values" or something like that, which is a better description of what is happening here, and also explains better why it isn't relevant for execute_many.